src/server/cluster/cluster_family.cc

```diff
@@ -33,6 +33,9 @@ ABSL_FLAG(std::string, cluster_node_id, "",
           "ID within a cluster, used for slot assignment. MUST be unique. If empty, uses master "
           "replication ID (random string)");
 
+ABSL_FLAG(uint32_t, migration_timeout, 30000,
```

Comment:
I would use 0 to disable this flag, if we have some bug in the logic and want to disable it

Reply:
I think it just makes the code more complicated. I would prefer leave as is